### DISCUSSION TITLE
Add a way to select drawables that are fully covered by others in the skin editor

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#22760] [Q&A] [answered]
With the addition of a much needed way to sort the depth of drawables within the skin editor manually, a lot of possibilities have be opened in the realm of skin creation but this has personally been a hindrance to me whilst making a complex skin

i suggest the selected `drawable` being changed upon further clicks ( so basically cycling through the drawables under your cursor) 

Here is an example skin to demonstrate how cluttered it could get: 

https://user-images.githubusercontent.com/74463310/222174980-ec10fd37-eec6-4f58-94b8-e48432fc4a5f.mov

layers ( not only for rulesets but for general use) would also aid with usability

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/22804
